Added a blog post call for testers for security updates (#295)
AlmaLinux/almalinux.org: content/blog/testers-needed-for-updated-packages-intel-and-amd-cpus-affected.md

---
title: "Testers needed for security updates: Intel and AMD CPUs affected"
type: blog
author: 
 name: "Andrew Lukoshko"
 bio: "Release Engineering Lead"
 image: /users/alukoshko.jpg

> date: '2023-08-10'

Reviewer: ```suggestion
date: '2023-08-14'
```

post:
    title: "Testers needed for security updates: Intel and AMD CPUs affected"
    image: /blog-images/testers-post.png
---

Earlier this week a few vulnerabilities have been reported to affect Intel and AMD CPUs. AlmaLinux OS Foundation calls for the community to help to test updated packages for AlmaLinux OS 8 and AlmaLinux OS 9.

### How do I install updated  packages?

> As AlmaLinux OS has changed its build processes in order to be [ABI compatible](https://almalinux.org/blog/future-of-almalinux/) with Red Hat, **Testing** repo has been created for packages that differ from RHEL and require additional testing by the community. 

Reviewer: ```suggestion
 As we recently changed the build process for AlmaLinux OS, and are now aiming to be [ABI compatible](https://almalinux.org/blog/future-of-almalinux/) with Red Hat Enterprise Linux, we are now able to release updates without waiting for the patches to be released upstream. To help accommodate that, a **Testing** repo has been created for packages that differ from RHEL and require additional testing by our users. 
```

To be able to help with testing, **Testing** repo should be enabled on the AlmaLinux machine:

```bash
dnf install -y almalinux-release-testing
```

> It's highly recommended to enable **Testing** repo on all non-production machines to participate in AlmaLinux development.

Reviewer: ```suggestion
We highly recommend and encourage you to enable the **Testing** repo on all non-production machines to expand participation in the AlmaLinux development.
```

### CVE-2022-40982 aka Downfall (Intel)

[CVE-2022-40982](https://downfall.page/) vulnerability is related to a Gather Data Sampling (GDS) transient execution side-channel vulnerability affecting Intel CPUs. This may allow an attacker to access stale data from previously used vector registers on the same physical core. Computing devices based on Intel Core processors from the 6th Skylake to (including) the 11th Tiger Lake generation are affected.

> The vulnerability can be mitigated by updating CPU microcode - `microcode_ctl` package:

Reviewer: ```suggestion
Once you enable the **Testing** repo, the vulnerability can be mitigated by updating CPU microcode - `microcode_ctl` package:
```

 
```bash
dnf update microcode_ctl
```

**To check that the installation is completed successfully, you can run:**

```bash
rpm -qa microcode_ctl
```

**Make sure that you've got the following version:**
* AlmaLinux OS 8 - microcode_ctl-20220809-2.20230808.1.el8_8.alma
* AlmaLinux OS 9 - microcode_ctl-20220809-2.20230808.1.el9_2.alma

**To update the CPU microcode run the following:**

```bash
echo 1 > /sys/devices/system/cpu/microcode/reload
```

### CVE-2023-20569 (AMD)

> [CVE-2023-20569](https://nvd.nist.gov/vuln/detail/CVE-2023-20569) vulnerability affects "Zen 3" and "Zen 4" AMD CPUs as it may allow an attacker to influence the return address prediction. This may potentially lead to information disclosure.

Reviewer: ```suggestion
The recently announced [CVE-2023-20569](https://nvd.nist.gov/vuln/detail/CVE-2023-20569) vulnerability affects "Zen 3" and "Zen 4" AMD CPUs as it may allow an attacker to influence the return address prediction. This may potentially lead to information disclosure.
```

> The vulnerability can be partially mitigated by updating `linux-firmware` package:

Reviewer: ```suggestion
Once you enable the **Testing** repo, the vulnerability can be partially mitigated by updating `linux-firmware` package:
```

```bash
dnf update linux-firmware
```

**To check that the installation is completed successfully, you can run:**

```bash
rpm -qa linux-firmware
```

**Make sure that you've got the following version:**
* AlmaLinux OS 8 - linux-firmware-20230404-114.git2e92a49f.el8_8.alma.1
* AlmaLinux OS 9 - linux-firmware-20230310-134.el9_2.alma.1

> **To update the CPU microcode run the following:**

Reviewer: ```suggestion
**To update the CPU microcode run the following or reboot the computer:**
```
